<launch>
<node name="ros0xrobot" pkg="ros0xrobot" type="ros0xrobotNode" output="screen">
<param name="port" value="/dev/ttyUSB0" />
<param name="CountsPerRev" value="2000" />
<param name="WheelDiameter" value="260.0" />
<param name="AxelLength" value="590.0" />
<param name="enableSonar" value="true" />
</node>

</launch>
